plugins/python/src/main/kotlin/io/paddle/plugin/python/dependencies/Dependencies.kt

```diff
@@ -0,0 +1,13 @@
+package io.paddle.plugin.python.dependencies
+
+class Dependencies {
+    private val _dependencies: MutableList<CachedPackage> = arrayListOf()
```

Comment:
Use just `dependencies` -- no need in `_`. Also, consider using standard constructors instead of `arrayListOf` and so on, those functions were created for varargs, not for creation of empty objects.